receipt_layoutlm/receipt_layoutlm/export_worker.py

```
@@ -0,0 +1,343 @@
+    """
+    sqs = boto3.client("sqs", region_name=region)
+
+    # Optional DynamoDB client for status updates
+    dynamo_client = None
+    if dynamo_table:
+        try:
+            from receipt_dynamo import DynamoClient
+
+            dynamo_client = DynamoClient(table_name=dynamo_table, region=region)
+            print(f"Connected to DynamoDB table: {dynamo_table}")
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)